sdk/eventhub/event-hubs/src/eventPosition.ts

```diff
@@ -121,8 +121,8 @@ export class EventPosition {
    * @returns EventPosition
    */
   static fromOffset(offset: number, isInclusive?: boolean): EventPosition {
-    if (offset == undefined) {
-      throw new Error('Missing parameter "offset"');
+    if (typeof offset !== "number" && typeof offset !== "string") {
```

Comment:
Why do we want to allow string?

Reply:
For latest, we would expect user to use the static helper `EventPosition.latest()`

Reply:
The reason to allow string was in case the offset was a number provided as a string. We had tests for this use-case already in receiver, but I can make it support strictly numbers.